Scripts/DowloadPortalItemData.swift

```diff
@@ -218,7 +219,7 @@ let portalItems: [PortalItem] = {
         // Omit the decoding errors from samples that don't have dependencies.
         let sampleDependencies = sampleJSONs
             .compactMap { try? parseJSON(at: $0) }
-        return sampleDependencies.flatMap(\.offlineData)
+        return Set(sampleDependencies.flatMap(\.offlineData))
```

Comment:
Flat mapping creates an intermediate array. That can be avoided by lazily flat mapping.

```suggestion
        return Set(sampleDependencies.lazy.flatMap(\.offlineData))
```